geo-types/src/line_string.rs

```diff
@@ -5,7 +5,7 @@ use crate::{CoordNum, Coordinate, Line, Point, Triangle};
 use std::iter::FromIterator;
 use std::ops::{Index, IndexMut};
 
-/// An ordered collection of two or more [`Coordinate`]s, representing a
+/// An ordered collection of two or more [Coordinate]s, representing a
```

Comment:
Nonblocking and subjective, but how do you feel about keeping the inline code block formatting? At least to me, it better indicates that "Coordinate" in this context is referring to a specific geo-types API

```suggestion
/// An ordered collection of two or more [`Coordinate`]s, representing a
```

Reply:
Done for all types!